modular_skyrat/code/game/mecha/working/ripley.dm

```diff
@@ -0,0 +1,41 @@
+/obj/mecha/working/ripley/firefighter/killdozer
+	desc = "Autonomous Power Loader Unit. This model is refitted with additional thermal and kinetic protection. Capable of accepting illegal upgrades, as well as damaging government property."
+	name = "\improper APLU \"Killdozer\""
+	icon = 'modular_skyrat/icons/mecha/sometimesreasonablemenmustdounreasonablethings'
+	icon_state = "ripley"
+	opacity = 1
+	bumpsmash = 1
+	step_in = 2
+	step_energy_drain = 3
+	normal_step_energy_drain = 3
+	melee_energy_drain = 7
+	fast_pressure_step_in = 2
+	slow_pressure_step_in = 2
+	max_temperature = 80000
+	max_integrity = 275
+	resistance_flags = LAVA_PROOF | FIRE_PROOF | ACID_PROOF
+	lights_power = 7
+	armor = list("melee" = 50, "bullet" = 55, "laser" = 40, "energy" = 50, "bomb" = 80, "bio" = 100, "rad" = 100, "fire" = 100, "acid" = 100)
```

Comment:
why is it rad proof, and can have 65% melee armor with 3 plates...

Reply:
infact why is this better then most ds mechs?

Reply:
I should probably remove the radproofing. But the armor is just fine because i mean, it's a boogaloo ripley.